### PR TITLE
Add alacritty's default theme pre-0.13

### PR DESCRIPTION
**Pull request by cyqsimon**
Alacritty v0.13 changed the default colour scheme. So I thought I'd add v0.12 defaults as a config.

### CODE REVIEW

File: themes/alacritty_0_12.toml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+blue = "#556a7d"
+magenta = "#75617b"
+cyan = "#5b7d78"
+white = "#828482"
+
+[colors.hints]
+start = { foreground = "#1d1f21", background = "#e9ff5e" }
+end = { foreground = "#e9ff5e", background = "#1d1f21" }
+
+[colors.normal]
```
Comment (kchibisov) [inline themes/alacritty_0_12.toml:28]: Could we have normal above bright, the order is usually `normal`, `bright`, and then `dim`.

File: themes/alacritty_0_12.toml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+black = "#1d1f21"
+red = "#cc6666"
+green = "#b5bd68"
+yellow = "#f0c674"
+blue = "#81a2be"
+magenta = "#b294bb"
+cyan = "#8abeb7"
+white = "#c5c8c6"
+
+[colors.primary]
```
Comment (kchibisov) [inline themes/alacritty_0_12.toml:38]: Should be first section.

**Review by kchibisov — DISMISSED** [review]

**Comment by kchibisov** [comment]
I'm not sure that I like the name `0_12`, maybe `alacritty_classic` should be better?

**Comment by cyqsimon** [comment]
> I'm not sure that I like the name `0_12`, maybe `alacritty_classic` should be better?

I used this name because it's a clearer demarkation. Besides, there's a non-zero chance that the default theme gets changed again in the future, and then it would become confusing.

File: themes/alacritty_0_12.toml (hunk 1 above)
Comment (cyqsimon) [reply themes/alacritty_0_12.toml]: Done. I had the sections sorted alphabetically previously.

**Comment by kchibisov** [comment]
nah, it won't change, unless both maintainers change.

**Comment by kchibisov** [comment]
anyway, it's up to @chrisduerr how to name the old variant.

**Review by chrisduerr — APPROVED** [review]
I like the name.

**Comment by moto-ctrl** [comment]
great stuff, I like the old theme, thanks for that!

**Pull request merged**